src/test/java/com/aws/greengrass/mqtt/bridge/MQTTBridgeTest.java

```diff
@@ -332,62 +334,86 @@ void GIVEN_Greengrass_with_mqtt_bridge_WHEN_invalid_mqttTopicMapping_updated_THE
         });
         assertThat(topicMapping.getMapping().size(), is(equalTo(0)));
 
-        Assertions.assertTrue(bridgeErrored.await(TEST_TIME_OUT_SEC, TimeUnit.SECONDS));
+        assertTrue(bridgeErrored.await(TEST_TIME_OUT_SEC, TimeUnit.SECONDS));
     }
 
     @Test
-    void GIVEN_Greengrass_with_mqtt_bridge_WHEN_CAs_updated_THEN_KeyStore_updated(ExtensionContext extensionContext) throws Exception {
-        ignoreExceptionOfType(extensionContext, InterruptedException.class);
-        serviceFullName = MQTTBridge.SERVICE_NAME;
-        initializeMockedConfig();
-        TopicMapping mockTopicMapping = mock(TopicMapping.class);
-        MessageBridge mockMessageBridge = mock(MessageBridge.class);
-        PubSubIPCEventStreamAgent mockPubSubIPCAgent = mock(PubSubIPCEventStreamAgent.class);
-        Kernel mockKernel = mock(Kernel.class);
-        MQTTClientKeyStore mockMqttClientKeyStore = mock(MQTTClientKeyStore.class);
-        MQTTBridge mqttBridge;
-        LocalMqttClientFactory localMqttClientFactory = new LocalMqttClientFactory(mockMqttClientKeyStore, ses);
+    void GIVEN_mqtt_bridge_WHEN_cda_ca_conf_changed_THEN_bridge_keystore_updated() throws Exception {
+        startKernelWithConfig("config.yaml");
```

Comment:
starting a kernel makes this an integration test. Try and keep this as a unit test.

Reply:
yeah, since the tests involve install/startup/shutdown lifecycle, i figured int test would give better assurances than mocking.  can revert it back to unit test though

Reply:
yes please. Having an integ test in addition is fine, but it should go with the other integ tests.

Reply:
yeah, we need a refactor, cause there's no integ folder for bridge yet.  they're just in this `MQTTBridgeTest` class